src/Model/LoginSession.php

```diff
@@ -229,15 +233,18 @@ public function getFriendlyUserAgent(): string
         $parser = Parser::create();
         $result = $parser->parse($this->UserAgent);
 
-        return sprintf('%s on %s', $result->ua->family, $result->os->toString());
+        return _t(
+            __CLASS__ . '.BROWSER_ON_OS',
+            "{browser} on {os}.",
+            ['browser' => $result->ua->family, 'os' => $result->os->toString()]
+        );
     }
 
     /**
      * @param Member|null $member
      * @param HTTPRequest|null $request
-     * @return LoginSession|null
      */
-    public static function getCurrentLoginSession(Member $member = null, HTTPRequest $request = null)
+    public static function getCurrentLoginSession(Member $member = null, HTTPRequest $request = null): ?self
     {
         if (!$member) {
             $member = Security::getCurrentUser();
```

Comment:
The `$member` variable is currently unused.  We should probably just remove `$member` from the function entirely?